packaging/rpm/temboard-agent.spec

```diff
@@ -4,7 +4,7 @@
 
 %{!?python_sitelib: %global python_sitelib %(%{__python2} -c "from distutils.sysconfig import get_python_lib; print (get_python_lib())")}
 
-%if 0%{?rhel} >= 8
+%if 0%{?rhel} >= 7
   %global __python /usr/bin/python3
 %endif
```

Comment:
This block could be dropped and we should just use the standard `__python3` macro.